build.sbt

```diff
@@ -10,7 +10,7 @@ val circeVersion = "0.13.0"
 val sttpVersion = "3.3.15"
 val Specs2Version = "4.6.0"
 val artifactory = "https://cognite.jfrog.io/cognite/"
-val cogniteSdkVersion = "1.5.20"
+val cogniteSdkVersion = "1.5.22-SNAPSHOT"
```

Comment:
TODO: update this once https://github.com/cognitedata/cognite-sdk-scala/pull/442 is merged